.babelrc

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
- [Optional] The project requirements demanded you use gitflow for this project, which means once you are done building your application on the dev branch, you should creat a pull request from the dev branch to the main branch and submit that pull request for review. Kindly note that please in all future work.